File: java/MostraData.java
/*
 * Programa Java demonstra como recuperar a data atual (máquina).
 *
 * Módulo: Lógica de programação
 * Author: Eder Magalhães
 */

import java.util.Date;

public class MostraData {

    public static void main(String[] args) {
        Date hoje = new Date();

        System.out.printf("Data (original): %s %n", hoje);
        System.out.printf("Data formatada: %1$te/%1$tm/%1$tY %n", hoje);
        System.out.printf("Data outro formato: %tD %n", hoje);
    }

}

File: python/mostraData.py
# -*- coding: utf-8 -*-

# Programa Python demonstra recuperar a data atual (máquina).
#
# Acesse e execute esse código Python em: http://pythonfiddle.com/mostra-data-ap/
#
# Módulo: Lógica de programação
# Author: Eder Magalhães

from datetime import datetime;

hoje = datetime.now()

print "Data (original): %s" % hoje;
print "Data formatada: %s" % hoje.strftime("%d/%m/%Y");
print "Data outro formato: %s" % hoje.strftime("%m/%d/%y");

